### PR TITLE
docs(how-to): marimo integration

### PR DESCRIPTION
**Pull request by mscolnick**
Many of our users @ marimo use Ibis and have strong integration with Ibis tables. We would love to be a part of your ecosystem page getting people set up to use Ibis in marimo

### CODE REVIEW

**Comment by lostmygithubaccount** [comment]
@mscolnick could you update to using sentence case per [the style and formatting guide](https://pr-10392-108024ad99082ee219bc8f4af6c828405c65a7a2--ibis-quarto.netlify.app/contribute/03_style#text-formatting)? looks good to me otherwise, thanks for the PR! (might have one other nit)

File: docs/how-to/visualization/marimo.qmd [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+            avg_bill_length=ibis._.bill_length_mm.mean(),
+            avg_bill_depth=ibis._.bill_depth_mm.mean(),
+        )
+    ),
+})
+```
+
+## Examples
+
+For more examples, checkout the Ibis notebooks in the marimo repo: <https://github.com/marimo-team/marimo/tree/main/examples/third_party/ibis>
```
Comment (lostmygithubaccount) [inline docs/how-to/visualization/marimo.qmd:94]:
```suggestion
For more examples, checkout the Ibis notebooks in the marimo repo: <https://github.com/marimo-team/marimo/tree/main/examples/third_party/ibis>.
```

File: docs/how-to/visualization/marimo.qmd [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+table
+```
+
+You can use the `.value` attribute to get the underlying selection in Python as an Ibis table. This value can be used to create visualizations, filters, and other interactive elements.
+
+```python
+table.value
+```
+
+### Interactive Filtering
```
Comment (cpcloud) [inline docs/how-to/visualization/marimo.qmd:35]:
```suggestion
### Interactive filtering
```

File: docs/how-to/visualization/marimo.qmd [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+Use marimo's no-code dataframe transformer to interactively filter and transform your Ibis data:
+
+```python
+transof = mo.ui.dataframe(penguins)
+transof
+```
+
+The `.value` attribute of the transformer contains the underlying Ibis table.
+
+### Combining Filters with Ibis Queries
```
Comment (cpcloud) [inline docs/how-to/visualization/marimo.qmd:46]:
```suggestion
### Combining filters with Ibis queries
```

File: docs/how-to/visualization/marimo.qmd [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+# Apply the filters to the Ibis query
+filtered_data = penguins.filter(
+    [penguins.bill_length_mm <= bill_length_slider.value]
+)
+
+# Display the filtered results in an interactive table
+mo.ui.table(filtered_data)
+```
+
+### Layout and Presentation
```
Comment (cpcloud) [inline docs/how-to/visualization/marimo.qmd:73]:
```suggestion
### Layout and presentation
```

**Review by cpcloud — APPROVED** [review]
Just addressing some of the style feedback.

**Comment by mscolnick** [comment]
Thanks @cpcloud for the fixes! Apologies I didn’t get to them yesterday

**Pull request merged**